src/presentation/test/integration/registration/generalPartner/add-general-partner-person.test.ts

```diff
@@ -46,6 +46,7 @@ describe("Add General Partner Person Page", () => {
         "limitedPartner",
         "dateEffectiveFrom"
       ]);
+      expect(res.text).toContain("WELSH - I confirm that the general partner is not disqualified under the directors disqualification legislation, as defined in the Limited Partnership Act 1907.");
```

Comment:
rather than check for this string in each test, can you just check if it contains the value from the translations file.

Reply:
you should be able to do it similar to the page title check a few lines above:
`expect(res.text).toContain(cyTranslationText.generalPartnersPage.disqualificationStatement);`

and change cyTranslationText to enTranslationText for the english tests

Reply:
Also this and the negative of the is a lot more readable.

Reply:
If we start doing this approach we'll be duplicating our static text lots of times throughout the tests, see what the consenus is, @dlloyd1-ch @mwestacott

Reply:
Alright added a testTranslations for the section containing the disqualification text.